### ISSUE TITLE
Docker Desktop - 2 minutes is not enough time to wait for Kubernetes cluster to start

### ISSUE THREAD

**Issue by TimothyLoyer** [tilt-dev/ctlptl#139]
While #134 fixed the issue where ctlctl stopped waiting for the cluster too early, I'm finding that 2 minutes is sometimes still not long enough. The connection on my Windows VM reports 140 Mbps download speeds (via fast.com), but still frequently takes closer to 4-5 minutes than the 2 minutes ctlptl will wait (maybe the bottleneck isn't internet, but we both have 32 GB of RAM and at least 8 reasonably fast CPU cores). I can of course just run the command a second time and it will work (because the cluster is usually up by then), but I've been surprised by how long it takes Docker Desktop to setup Kubernetes, both on my Windows test VM and my co-worker's Mac.

```
# ctlptl apply -f ./cluster.yaml
Applied new Docker Desktop settings. Waiting 2m for Docker Desktop to restart...
Switched to context "docker-desktop".
Waiting 2m for Kubernetes cluster "docker-desktop" to start...
timed out waiting for cluster to start: context deadline exceeded
```

**Comment by TimothyLoyer** [comment]
I just timed it on Windows, and it took about 3m 20s to start the cluster.

**Comment by nicks** [comment]
ahhhhh ya I suspect the bottleneck is how fast it can download the kubernetes images

i don't want to make it TOO long, because I have definitely seen situations where docker-desktop's kubernetes got wedged and would never start up, and would need to be restarted.

**Comment by nicks** [comment]
should be fixed in v0.5.7! let me know if the timeouts are still causing problems, we can do as many bugfix releases as we need to get it right!
